modules/google-tag-manager/bus/events/AddToCart.ts

```diff
@@ -0,0 +1,6 @@
+import useProduct from '#ioc/composables/useProduct'
+
+export default interface AddToCart {
```

Comment:
If you move events from google-analytics to proper modules, they wont have to be duplicated here.